test/new-e2e/runner/parameters/caching_store.go

```diff
@@ -20,11 +22,13 @@ func newCachingStore(s valueStore) valueStore {
 	}
 }
 
-func (s *cachingStore) get(key string) (string, error) {
+func (s *cachingStore) get(key StoreKey) (string, error) {
 	s.l.Lock()
 	defer s.l.Unlock()
 
-	value, found := s.cache[key]
+	cachingStoreKey := string(key)
```

Comment:
Note: You can change `cache map[string]string` to `cache map[StoreKey]string` in order to avoid this conversion.